Src/Plugins/Security/Security.csproj

```diff
@@ -44,11 +45,18 @@
   </ItemGroup>
 
   <ItemGroup>
+    <Reference Include="aliyun-net-sdk-iot">
+      <HintPath>ExternalDLLs\aliyun-net-sdk-Iot.dll</HintPath>
```

Comment:
>ExternalDLLs\aliyun-net-sdk-Iot.dll [](start = 16, length = 35)

check how we do with ocktokit. if u can, do the same.

Reply:
Done

---
In reply to: [591818461](https://github.com/microsoft/sarif-pattern-matcher/pull/285#discussion_r591818461) [](ancestors = 591818461)